mysql/datadog_checks/mysql/mysql.py

```diff
@@ -147,6 +152,24 @@ def check_performance_schema_enabled(self, db):
 
         return self.performance_schema_enabled
 
+    def _check_events_wait_current_enabled(self, db):
+        if not self._check_performance_schema_enabled(db):
+            self.log.debug('`performance_schema` is required to enable `events_waits_current`')
+            return
+        if self.events_wait_current_enabled is None:
+            with closing(db.cursor()) as cursor:
+                cursor.execute(
+                    """\
+                    SELECT
+                        NAME,
+                        ENABLED
+                    FROM performance_schema.setup_consumers WHERE NAME = 'events_waits_current'
+                    """
+                )
```

Comment:
This is a nit that is out of scope for this PR because the pattern already exists, but for the future I think it would be better to make this a boolean function and change the query to:

```
 cursor.execute(
                    """\
                    SELECT count(*)
                    FROM performance_schema.setup_consumers WHERE NAME = 'events_waits_current'
                    AND ENABLED = 'YES'
                    """
                )
return cursor.fetchone()[0] > 0

```

This would avoid leaking state and making the monolithic class more complex with additional public variables.